src/helpers/determineOutcome.ts

```diff
@@ -1,26 +1,38 @@
-import { RuleConfig, RuleResult } from '@frmscoe/frms-coe-lib/lib/interfaces';
+import {
+  Band,
+  RuleConfig,
+  RuleResult,
+} from '@frmscoe/frms-coe-lib/lib/interfaces';
 
 const determineOutcome = (
   value: number,
   ruleConfig: RuleConfig,
   ruleResult: RuleResult,
 ): RuleResult => {
-  if (value || value === 0) {
-    for (const band of ruleConfig.config.bands) {
-      if (
-        (!band.lowerLimit || value >= band.lowerLimit) &&
-        (!band.upperLimit || value < band.upperLimit)
-      ) {
-        ruleResult.subRuleRef = band.subRuleRef;
-        ruleResult.result = band.outcome;
-        ruleResult.reason = band.reason;
-        break;
-      }
+  const { bands /*, parameters, case: ruleCase */ } = ruleConfig.config;
```

Comment:
Why this notation instead of 
```suggestion
  const bands /*, parameters, case: ruleCase */ = ruleConfig.config.bands;
```

Reply:
Purely cosmetic (and less typing for the lazy :D). It's mainly so we don't repeat ourselves in the future since this function will also need to use `case`.

Ultimately, it's:
```js
const {bands, parameters} = ruleConfig.config;
```

vs

```js
const bands = ruleConfig.config.bands
const parameters = ruleConfig.config.parameters
....
```
Do you prefer the latter?